install/install.qmd

```diff
@@ -105,90 +106,136 @@ For more information, see the
 
 <!-- ---------- CmdStan ---------- -->
 ::::: {.install #install-cmdstan-github-rel .hidden}
-TODO 1
+Download a release from GitHub:  https://github.com/stan-dev/cmdstan/releases.
+
+Then follow [these instructions to build CmdStan](https://mc-stan.org/docs/cmdstan-guide/installation.html#building-cmdstan){target="_blank"}.
 :::::
 ::::: {.install #install-cmdstan-conda .hidden}
-TODO 2
+Run `conda install -c conda-forge cmdstan`.
+
+Note: this will also install CmdStan and any system prerequisites.
 :::::
 ::::: {.install #install-cmdstan-github-src .hidden}
-TODO 3
-:::::
-<!-- ---------- Pystan ---------- -->
-::::: {.install #install-pystan-pip .hidden}
-TODO 4
-:::::
-::::: {.install #install-pystan-conda .hidden}
-TODO 5
-:::::
-::::: {.install #install-pystan-github-src .hidden}
-TODO 6
+Run `git clone https://github.com/stan-dev/cmdstan.git --recursive`
+
+This will download the source code from the current development branch of CmdStan into a directory named cmdstan,
+along with the submodules for core Stan code and the Stan math library.
+
+Then follow [these instructions to build CmdStan](https://mc-stan.org/docs/cmdstan-guide/installation.html#building-cmdstan){target="_blank"}.
 :::::
 <!-- ---------- RStan ---------- -->
 ::::: {.install #install-rstan-cran .hidden}
-TODO 7
+In R, run `install.packages("rstan")`
+
+For more information, see the
+[RStan Getting Started wiki](https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started){target="_blank"}
 :::::
 ::::: {.install #install-rstan-runiverse .hidden}
-TODO 8
+In R, run `install.packages("rstan", repos = c('https://stan-dev.r-universe.dev', getOption("repos")))`.
+
+For more information, see the
+[RStan Getting Started wiki](https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started){target="_blank"}
 :::::
 ::::: {.install #install-rstan-conda .hidden}
 Run `conda install -c conda-forge r-rstan`.
 
-Note: this will also install any system prerequesites.
+Note: this will also install any system prerequisites.
 :::::
 ::::: {.install #install-rstan-github-src .hidden}
-TODO 9
+Run `git clone https://github.com/stan-dev/rstan.git --recursive`
+
+For more information, see the
+[RStan Getting Started wiki](https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started){target="_blank"}
 :::::
 <!-- ---------- Stan.jl ---------- -->
 ::::: {.install #install-stan.jl-julia-pkg .hidden}
-TODO 10
+To install Stan.jl e.g. in the Julia REPL: `] add Stan`,
+then use Conda.jl or conda to install CmdStan.
+
+See [Stan.jl documentation](http://stanjulia.github.io/Stan.jl/v6.0/INSTALLATION.html) for further details.
 :::::
 ::::: {.install #install-stan.jl-github-src .hidden}
-TODO 11
+Run `https://github.com/StanJulia/Stan.jl.git`,
+then follow instructions in the README file.
```

Comment:
This seems like it is missing some steps.

In general the way to do this in Julia is to clone the repository, launch Julia from inside it, and then `] dev .`